packages/ssz/src/types/composite/container.ts

```diff
@@ -248,7 +248,8 @@ export class ContainerType<T extends ObjectLike = ObjectLike> extends CompositeT
     const value = {} as T;
     for (const [fieldName, fieldType] of Object.entries(this.fields)) {
       const expectedCase = options && options.case;
-      const expectedFieldName = toExpectedCase(fieldName, expectedCase);
+      const customCasingMap = options && options.casingMap;
+      const expectedFieldName = toExpectedCase(fieldName, expectedCase, customCasingMap);
```

Comment:
Should we add this json options as an option to the container itself too? That would make declaration easier such that you define the fields and the casing in one place

Reply:
i am in favor of that, especially when there is container nesting (and it will be more optimized as the lookup will be in the container fields local scope) . you want to do it in this PR or separate?

Reply:
Good point, do it a another one :+1: